tools/stern/src/Stern/API.hs

```diff
@@ -581,6 +590,10 @@ deleteTeam (givenTid ::: email) = do
     handleNoTeam = ifNothing (mkError status404 "no-binding-team" "No such binding team")
     wrongMemberCount = mkError status403 "wrong-member-count" "Only teams with 1 user can be deleted"
     bindingTeamMismatch = mkError status404 "binding-team-mismatch" "Binding team mismatch"
+deleteTeam (tid ::: True ::: _) = do
+  void $ Intra.getTeamData tid -- throws 404 if team does not exist
```

Comment:
```suggestion
```

(`deleteBindingTeamForce` does that, too, no?)

Reply:
unfortunately errors from the internal API are not propagated to stern.
instead stern responds with 500
`deleteBindingTeamForce` responds with 404 but this will be a 500 in stern

Reply:
Ok.  I would have preferred to fix that (and re-throw the error) rather than adding this awkward workaround, but it's not essential.

Reply:
me too, but I thought I stick to the convention for now